composer.json

```diff
@@ -3,11 +3,11 @@
   "description": "PrestaShop coding standards",
   "license": "MIT",
   "require": {
-    "php": ">=7.2.5",
-    "symfony/console": "~3.2 || ~4.0 || ~5.0",
+    "php": "^8.0 || ^7.2",
```

Comment:
Hi @Quetzacoalt91, thx for the PR
You seem authorised the following version:
-7.2.0
-7.2.1
-7.2.2
-7.2.3
-7.2.4

It was not the case before youre PR

Reply:
Well, I think it would be safer to provide each minor version we are compatible with, so this library becomes compliant with a future PHP 9 when we're ready.
From my understanding putting `^7.2.5` would make only this version compliant with versions >= `7.2.5` and < `7.3.0`.

I reverted it, I'm also fine with the current line.